### PR TITLE
[pt] Added/improved APs in rule ID:GENERAL_NUMBER_AGREEMENT_ERRORS

### PR DESCRIPTION
**Pull request by marcoagpinto**
Heya, @susanaboatto and @p-goulart ,

I have added an AP and fixed/improved two others.

## Summary by CodeRabbit

- **New Features**
	- Introduced a new antipattern for the phrase "para tal" to improve grammar detection in Portuguese.

- **Improvements**
	- Enhanced accuracy of grammar checking by normalizing postag patterns across multiple tokens.

### CODE REVIEW

File: languagetool-language-modules/pt/src/main/resources/org/languagetool/rules/pt/grammar.xml [hunk 1]
```diff
@@ -4248,20 +4248,27 @@ Foundation, Inc., 51 Franklin St, Fifth Floor, Boston, MA 02110-1301, USA.
             <!--
            MARCOAGPINTO inserted global antipatterns for all subrules on top using the logic (2-MAR-2023+)
       -->
+            <antipattern>
+                <token>para</token>
+                <token>tal</token>
+                <token postag='(SPS00:)?[DP]...[PN].+|VM...P.+' postag_regexp='yes'/>
+                <token postag="AQ..[PN].+|N..[PN].+|Z0.[PN].+|(SPS00:)?[DP]...[PN].+|VM...P.+" postag_regexp="yes"/>
+                <example>Quero para tal esses dois telemóveis.</example>
```
Comment (susanaboatto) [inline languagetool-language-modules/pt/src/main/resources/org/languagetool/rules/pt/grammar.xml:4256]: Could also work with "para tanto"

Comment (marcoagpinto) [reply languagetool-language-modules/pt/src/main/resources/org/languagetool/rules/pt/grammar.xml:4256]: @susanaboatto 

Done!

**Comment by marcoagpinto** [comment]
@susanaboatto 

Any feedback on this one?

Thanks!

**Comment by marcoagpinto** [comment]
Heya, @p-goulart and @susanaboatto 

LanguageTool entered feature freeze today, could one of you approve these changes before the official release?

Thanks!

**Review by susanaboatto — APPROVED** [review]

**Pull request merged**